astrokat/observe_main.py

```diff
@@ -283,8 +283,8 @@ def subarray_setup(self, instrument):
             return
         approved_sb_sensor_value = approved_sb_sensor.get_value()
         if self.array.sb_id_code not in approved_sb_sensor_value:
-            user_logger.info('Skipping instrument checks - {} not in approved_schedule'
-                             .format(self.array.sb_id_code))
+            user_logger.info('Skipping instrument checks - {} not in approved_schedule'.format(
```

Comment:
Tip: loggers like the old-school formatting more, since you can go:
```Python
user_logger.info('Skipping instrument checks - %s not in approved_schedule', self.array.sb_id_code)
```
(plus the string is only formatted if the log message is actually emitted, saving a small amount of effort)